lean4-infoview/package.json

```diff
@@ -3,34 +3,35 @@
   "version": "0.1.0",
   "description": "An interactive display for the Lean 4 theorem prover",
   "scripts": {
-    "watch": "webpack --env development --watch",
-    "watchTest": "webpack --env development --watch",
-    "build": "webpack"
+    "watch": "rollup --config --environment NODE_ENV:development --watch",
+    "watchTest": "npm run watch",
+    "build": "rollup --config --environment NODE_ENV:production"
   },
-  "browser": "dist/index",
+  "browser": "dist/index.production.min.js",
   "types": "dist/index",
   "files": [
     "dist/*"
   ],
   "license": "Apache-2.0",
   "devDependencies": {
-    "@types/react": "^17.0.38",
-    "@types/react-dom": "^17.0.11",
-    "css-loader": "^6.5.1",
-    "style-loader": "^3.3.1",
-    "ts-loader": "^9.2.6",
-    "typescript": "^4.5.4",
-    "vscode-languageserver-protocol": "^3.17.0-next",
-    "webpack": "^5.65.0",
-    "webpack-cli": "^4.9.1"
+    "@rollup/plugin-commonjs": "^21.0.2",
+    "@rollup/plugin-node-resolve": "^13.1.3",
+    "@rollup/plugin-replace": "^4.0.0",
+    "@rollup/plugin-typescript": "^8.3.1",
+    "@rollup/plugin-url": "^6.1.0",
+    "@types/react": "^17.0.39",
+    "@types/react-dom": "^17.0.13",
+    "react-popper": "^2.2.5",
+    "rollup": "^2.70.0",
+    "rollup-plugin-css-only": "^3.1.0",
+    "rollup-plugin-terser": "^7.0.2",
+    "typescript": "^4.6.2"
   },
   "dependencies": {
     "@lean4/infoview-api": "^0.1.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "@vscode/codicons": "0.0.28",
     "react-fast-compare": "^3.2.0",
-    "react-popper": "^2.2.5",
     "tachyons": "^4.12.0",
-    "vscode-codicons": "0.0.17"
+    "vscode-languageserver-protocol": "^3.17.0-next"
```

Comment:
AFAICT we only use this for type-imports now?  Is there a reason it moved to `dependencies`?

Reply:
I marked the type-imports to distinguish runtime-imports, of which there are two, [here](https://github.com/leanprover/vscode-lean4/blob/0abfd048b8937446a8ff7677a9c75b7432bc7cee/lean4-infoview/src/infoview/infos.tsx#L28) and [here](https://github.com/leanprover/vscode-lean4/blob/0abfd048b8937446a8ff7677a9c75b7432bc7cee/lean4-infoview/src/infoview/messages.tsx#L25). So currently it does get bundled in.